My NixOS config uses 12 cores by default, and since I tried to start the PR I'm getting errors like 
```
/nix/store/8x3b9g3hjyw1fdkfry01yhp8lkccnfi7-x86_64-w64-mingw32-binutils-2.38/bin/x86_64-w64-mingw32-dlltool: dlls/iphlpapi/libiphlpapi.delay.a: error reading iphlpapi_dll_s00001.o: No such file or directory
winebuild: /nix/store/8x3b9g3hjyw1fdkfry01yhp8lkccnfi7-x86_64-w64-mingw32-binutils-2.38/bin/x86_64-w64-mingw32-dlltool failed with status 1
make: *** [Makefile:96346: dlls/iphlpapi/libiphlpapi.delay.a] Error 1
make: *** Waiting for unfinished jobs....
/nix/store/8x3b9g3hjyw1fdkfry01yhp8lkccnfi7-x86_64-w64-mingw32-binutils-2.38/bin/x86_64-w64-mingw32-dlltool: dlls/iphlpapi/libiphlpapi.cross.a: error reading iphlpapi_dll_t.o: No such file or directory
winebuild: /nix/store/8x3b9g3hjyw1fdkfry01yhp8lkccnfi7-x86_64-w64-mingw32-binutils-2.38/bin/x86_64-w64-mingw32-dlltool failed with status 1
make: *** [Makefile:96346: dlls/iphlpapi/libiphlpapi.cross.a] Error 1
```

When I build wine 7.4 off its original commit dd6c7096d629d9c6814c6980180254d3336a9a3b I see no such problem (35619ce1e513fa044b341f62a633deaa458eed06 is still okay as well). But on current master (without this PR) I do get it. It's been a huge pain trying to bisect the problem because of the long compile times.

Setting `enableParallelBuilding = false`; makes it go away, but that will make rebuilding the wine variants take way too much time.

--- a/pkgs/applications/emulators/wine/base.nix
+++ b/pkgs/applications/emulators/wine/base.nix
@@ -175,7 +175,9 @@ stdenv.mkDerivation ((lib.optionalAttrs (buildScript != null) {
     done
   '';
 
-  enableParallelBuilding = true;
+  # Until https://github.com/NixOS/nixpkgs/pull/172617 is applied,
+  # parallel builds do not always work because of a bug in dlltool.
+  enableParallelBuilding = false;
 
   # https://bugs.winehq.org/show_bug.cgi?id=43530
   # https://github.com/NixOS/nixpkgs/issues/31989

